pkg/pillar/types/global.go

```diff
@@ -689,25 +696,54 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.GlobalSettings = make(map[GlobalSettingKey]ConfigItemSpec)
 	configItemSpecMap.AgentSettings = make(map[AgentSettingKey]ConfigItemSpec)
 
-	configItemSpecMap.AddIntItem(ConfigInterval, 60, 5, 0xFFFFFFFF)
-	configItemSpecMap.AddIntItem(MetricInterval, 60, 5, 0xFFFFFFFF)
-	configItemSpecMap.AddIntItem(ResetIfCloudGoneTime, 7*24*3600, 120, 0xFFFFFFFF)
+	// timer.config.interval(seconds)
+	// MaxValue needs to be limited. If configured too high, the device will wait
+	// too long to get next config and is practically unreachable for any config
+	// changes or reboot through cloud.
+	configItemSpecMap.AddIntItem(ConfigInterval, 60, 5, HourInSec)
+	// timer.metric.interval (seconds)
+	// Need to be careful about max value. Cloud uses this to indicate if a
+	// device is online or suspect state.
+	configItemSpecMap.AddIntItem(MetricInterval, 60, 5, HourInSec)
+	// timer.reboot.no.network (seconds) - reboot after no cloud connectivity
+	configItemSpecMap.AddIntItem(ResetIfCloudGoneTime, 7*DayInSec,
+		120, 2*YearInSec)
+	// timer.update.fallback.no.network (seconds) - fallback after no cloud connectivity
+	// Max designed to allow the option of never rebooting even if device
+	//  can't connect to the cloud
```

Comment:
You are confusing the ResetIfCloudGone and FallbackIFCloudGone. The latter applies to the testing phase of an EVE image update. The former is for normal operation. I don't think we need to enforce a max for either one. Note that if FallbackIFCloudGone is set to larger than 10 minutes we are in untested territory but that doesn't mean we have a good idea about a max. I think the FallbackIFCloudGone should actually be removed once we have user end-to-end control whether or not to commit to a new EVE image.

Reply:
Fixed it..

Reply:
Hmm.. Good catch.. Fixed the comments.. Set the values back to max..